integreat_compass/cms/views/components/decline_list_view.py

```diff
@@ -0,0 +1,25 @@
+from django.views.generic import TemplateView
+
+from ...models import OfferVersion
+
+
+class DeclineListView(TemplateView):
+    """
+    View for the decline list
+    """
+
+    template_name = "components/decline_list.html"
+
+    def get_context_data(self, **kwargs):
+        r"""
+        Context for the list of declined offers
+
+        :param \**kwargs: The supplied keyword arguments
+        :type \**kwargs: dict
+
+        :return: The rendered template response
+        :rtype: ~django.template.response.TemplateResponse
+        """
+        return super().get_context_data(**kwargs) | {
+            "declined_offers": OfferVersion.objects.filter(state="False")
+        }
```

Comment:
```suggestion
        return super().get_context_data(**kwargs) | {
            "declined_offers": OfferVersion.objects.filter(state="False").distinct(
                "offer"
            )
        }
```

I think in the long run (not the MVP 😄) we should show the reports in the voting form if the previous public version received reports as well. Otherwise Providers might add an offer, it gets approved, it gets reported, Provider changes some small thing not relating to the report, offer gets approved again because the Board is not aware of the previous report.

An edit history might even be a good idea 😄